syntax = "proto3";

package document.intelligence.v1;

option go_package = "ztp/proto/services/document/intelligence";

service DocumentClassifierService {
  rpc ClassifyDocument(ClassifyDocumentRequest) returns (ClassifyDocumentResponse);
}

message ClassifyDocumentRequest {
  string document_id = 1;
  string job_id = 2;
  string image_url = 3;
}

message ClassifyDocumentResponse {}

enum DocumentType {
  DOCUMENT_TYPE_UNSPECIFIED = 0;
  DOCUMENT_TYPE_INVOICE = 1;
}
